deskpro/deskpro-admin-frontend
# Allows for functions with no side effects to be attached to actions

To test:

- Go to agents table page
- Check a row in the table
- Click actions
- Click menu item `No side effect action`
- See console log
